src/templates/blog-post.js

```diff
@@ -59,6 +59,7 @@ export default ({ data, path }) => {
 export const query = graphql`
   query($id: String!, $blogImages: [String]) {
     contentfulBlogPost(id: { eq: $id }) {
+      id
```

Comment:
the `id` is a random string from Gatsby that can change frequently, let's use `contentful_id` which is also shorter!

Reply:
Fixed, thanks!